pkg/commenter/github/commitFileInfo.go

```diff
@@ -40,7 +44,13 @@ func getCommitFileInfo(ghConnector *connector) ([]*commitFileInfo, error) {
 }
 
 func (cfi commitFileInfo) calculatePosition(line int) *int {
-	position := line - cfi.hunkStart
+	var ch chunkLines
+	for _, lines := range cfi.ChunkLines {
+		if line >= lines.Start && line <= lines.End {
```

Comment:
Move this if logic to a function named `Contains` on  `chunkLines` struct
Then you can reuse it in `checkIfLineInChunk`